lib/v2/gocn/topics.js

```diff
@@ -12,12 +13,18 @@ module.exports = async (ctx) => {
         },
     });
 
-    const list = response.data.data.list;
+    const items = response.data.data.articlePageList.list.map((item) => ({
+        title: item.name,
+        link: `${article_url}/s/${item.spaceGuid}/d/${item.guid}`,
+        description: item.name,
```

Comment:
`description` should use value from `item.content`

Reply:
I think `item.Content` is a very large piece of content. Much like the body of an article. That's why I didn't use `Content`.